che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/Messages.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - API 
+ *   SAP           - initial implementation
+ *******************************************************************************/
+package org.eclipse.che.git.impl.jgit;
+
+public class Messages {
```

Comment:
I agree with you, so I hard-coded values of messages and removed Messages and ExternalStringsSource classes